server/pkg/mail/healthcheck.go

```diff
@@ -3,6 +3,14 @@ package mail
 import (
 	"context"
 	"fmt"
+	"net/smtp"
+	"regexp"
+	"strconv"
+)
+
+const (
+	hostCheckRE = "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\\-]*[A-Za-z0-9])$"
+	portCheckRE = "^([1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$"
```

Comment:
why RE for number checking?

Reply:
This is for checking the Port Number [making sure it's between the range of [0 and 65535]